python/cac_tripplanner/destinations/views.py

```diff
@@ -58,7 +63,15 @@ def directions(request):
     :param request: Request object
     :returns: A rendered response
     """
-    return base_otp_view(request, 'directions.html')
+    return base_view(request, 'directions.html')
+
+
+def place_detail(request, pk):
+    destination = get_object_or_404(Destination.objects.published(), pk=pk)
+    more_destinations = Destination.objects.published().order_by('?').exclude(pk=destination.pk)[:3]
```

Comment:
Not something I remember off the top of my head, but I just tried in shell_plus and learned that, while index lookups get grumpy when they're out of range, slices are forgiving.